crates/virtual-dom-rs/tests/diff_patch.rs

```diff
@@ -179,5 +179,16 @@ fn text_root_node() {
         new: html! { New text },
         override_expected: None,
     }
-        .test();
+    .test();
+}
+
+#[wasm_bindgen_test]
+fn text_remove_last() {
+    DiffPatchTest {
+        desc: "Removing a text node works",
+        old: html! { <div>a<br></div> },
```

Comment:
Minor point, but this isn't really removing a text node but rather replacing it.

Because of that, I might've made the test:

Before: `<div>a</div>` 
After: `<div><br /></div>`

And description "Replacing a text node with an element".

But this looks good so I'll leave whether or not that's worth the time up to you!

Reply:
To clarify, the diff/patch algorithm doesn't remove `a`. It replaces `a` with `<br>` and then removes the second `<br>`

Reply:
Yep, you're right. I'll adjust the test.

Reply:
Done, the test should be more focussed now.